NICE.Identity.Test/UnitTests/Authorisation.WebAPI/Services/OrganisationsServiceTests.cs

```diff
@@ -46,6 +46,19 @@ public void Create_organisation()
             actualDateAddedCreatedOrganisation.ShouldBe(expectedDateAdded, TimeSpan.FromSeconds(5));
         }
 
+        [Fact]
+        public void Create_organisation_errors_when_already_exists()
+        {
+            //Arrange
+            var context = GetContext();
+            var organisationService = new OrganisationsService(context, _logger.Object);
+
+            organisationService.CreateOrganisation(new ApiModels.Organisation { Name = "ExistingOrg" });
+
+            //Act + Assert
+            Should.Throw<Exception>(() => organisationService.CreateOrganisation(new ApiModels.Organisation() { Name = "existingOrg" }));

```

Comment:
Should we be asserting on the actual exception text i.e. `that organisation already exists`, it's possible some other exception could be thrown. What do you think?